.circleci/config.yml

```diff
@@ -32,8 +32,8 @@ jobs:
       - run:
           name: Release the Astronomer chart
           command: |
-            printf "%s" $${GCP_TOKEN} > /tmp/gcs_token.json
```

Comment:
two issues here:
- printf command modifies the string to require a sed command to adjust (can't tell you why). instead use 'echo'
- two $$ is Makefile syntax, not normal script